analyzer/src/main/kotlin/managers/Conan.kt

```diff
@@ -0,0 +1,316 @@
+        return runInspectRawField(node["display_name"].textValueOrEmpty(), workingDir, field)
+    }
+
+    /**
+     * Project package options differ depending on which definition file is used:
+     * conanfile.py: allows us to run 'conan inspect conanfile.py' and extract useful project metadata
+     * conanfile.txt: does not allow 'conan inspect conanfile.txt'.
+     * TODO: Format of 'conan info' output of conanfile.txt may be such that we can get project metadata
+     *  from the 'requires' field... need to investigate whether this is a sure thing before implementing.
+     */
+    private fun extractProjectPackage(rootNode: JsonNode, definitionFile: File, workingDir: File): Package {
+        val projectPackageJson = rootNode.find {
+            it["reference"].textValueOrEmpty().contains(definitionFile.name)
+        }!!
+
+        if (definitionFile.name == "conanfile.py") {
+            // conanfile.py
+            return Package(
+                id = Identifier(
+                    type = managerName,
+                    namespace = "",
+                    name = runInspectRawField(definitionFile.name, workingDir, "name"),
+                    version = runInspectRawField(definitionFile.name, workingDir, "version")
+                ),
+                declaredLicenses = extractDeclaredLicenses(projectPackageJson),
+                description = runInspectRawField(definitionFile.name, workingDir, "description"),
+                homepageUrl = projectPackageJson["homepage"].textValueOrEmpty(),
+                binaryArtifact = RemoteArtifact.EMPTY, // TODO: implement me!
+                sourceArtifact = RemoteArtifact.EMPTY, // TODO: implement me!
+                vcs = VcsInfo(
+                    type = VcsType.GIT,
+                    url = projectPackageJson["url"].textValueOrEmpty(),
+                    revision = projectPackageJson["revision"].textValueOrEmpty()
+                )
+            )
+        } else {
+            // conanfile.txt
+            return Package(
+                id = Identifier(
+                    type = managerName,
+                    namespace = "",
+                    name = projectPackageJson["reference"].textValueOrEmpty(),
+                    version = ""
+                ),
+                declaredLicenses = extractDeclaredLicenses(projectPackageJson),
+                description = "",
+                homepageUrl = projectPackageJson["homepage"].textValueOrEmpty(),
+                binaryArtifact = RemoteArtifact.EMPTY, // TODO: implement me!
+                sourceArtifact = RemoteArtifact.EMPTY, // TODO: implement me!
+                vcs = VcsInfo(
+                    type = VcsType.GIT,
+                    url = projectPackageJson["url"].textValueOrEmpty(),
+                    revision = projectPackageJson["revision"].textValueOrEmpty()
+                )
+            )
+        }
+    }
+
+    /**
+     * Run 'conan install .' to install packages in specified directory
```

Comment:
Please explain in the code comment why you're not using `run` here and thus are accepting the program to eventually fail.